package.json

```diff
@@ -117,10 +117,6 @@
     }
   },
   "lint-staged": {
-    "*.md": [
-      "yarn gen-snapshot-tests",
-      "git add packages/*/__tests__/generated-snapshot.test.js"
-    ],
```

Comment:
Is there anything else that will generate these snapshot tests? or are they just turned off now?

Reply:
I think we could use husky to run our scripts on all files.  It would take a bit longer to run, but they would run in series and wouldn't affect each other.